hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableInputFormat.java

```diff
@@ -140,7 +116,7 @@ public CarbonTable getOrCreateCarbonTable(Configuration configuration) throws IO
 
     SegmentUpdateStatusManager updateStatusManager =
         new SegmentUpdateStatusManager(carbonTable, loadMetadataDetails);
-    List<Segment> invalidSegments = new ArrayList<>();
+    List<String> invalidSegments = new ArrayList<>();
```

Comment:
For all Segment change to String, change the variable name also

Reply:
done